cmd/handoff-bss-metadata.go

```diff
@@ -51,9 +51,9 @@ var (
 	dataFile      string
 	cloudInitData map[string]bssTypes.CloudInit
 	sshConfig     *ssh.ClientConfig
-	token         string
 
-	vlansToGather = []string{"bond0.nmn0"}
+	// TODO: SET THIS BACK
+	vlansToGather = []string{"vlan002"}
```

Comment:
Does his needs to be changed back now to work with 1.2?

Reply:
GREAT catch thank you so much for finding that!